docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -32,24 +32,26 @@ Prerequisites
 Software
 ~~~~~~~~
 
-- A :doc:`Flocker development machine <vagrant>`.
 - A web browser.
-- An up-to-date clone of the `Flocker repository <https://github.com/ClusterHQ/flocker.git>`_.
-- An up-to-date clone of the `homebrew-tap repository <https://github.com/ClusterHQ/homebrew-tap.git>`_.
 - `gsutil Python package <https://pypi.python.org/pypi/gsutil>`_ on your workstation.
+- `Vagrant`_ (1.6.2 or newer)
+- `VirtualBox`_
+
+.. _`Vagrant`: https://docs.vagrantup.com/
+.. _`VirtualBox`: https://www.virtualbox.org/
 
 Access
 ~~~~~~
 
 
-- Access to `Google Cloud Storage`_ using `gsutil`_ on your workstation and your :doc:`Flocker development machine <vagrant>`.
+- Access to `Google Cloud Storage`_ using `gsutil`_ on your workstation.
```

Comment:
This change seems wrong, you need gsutil on the vagrant-dev box in order to upload the wheel file that you've generated there.
And this change clashes with the changes below, which describe setting up gsutil on the vagrant-dev box.